ios/SnapCloneXcode/test_runner.sh

```
@@ -0,0 +1,34 @@
+echo '📱 Building SnapClone app...'
+xcodebuild build-for-testing     -project SnapClone.xcodeproj     -scheme SnapClone     -destination 'platform=iOS Simulator,name=iPhone 16,OS=18.6'     -configuration Debug
+
+echo '✅ Build completed successfully'
+
+# Launch the iOS Simulator
+echo '🚀 Launching iOS Simulator...'
+xcrun simctl boot 'iPhone 16' || echo 'Simulator already running'
+
+# Install the app
+APP_PATH=$(find ~/Library/Developer/Xcode/DerivedData -name 'SnapClone.app' -type d | head -1)
+if [ \! -z "$APP_PATH" ]; then
```

Shellcheck (0.10.0)

[warning] Unknown binary operator.

(SC2057)